orttraining/orttraining/test/gradient/gradient_checker.h

```diff
@@ -76,26 +76,35 @@ class GradientChecker {
       // because the gradient op does not handle the case. We have to use this flag
       // to disable check for not having gradient cases in order to pass those test.
       // Remove this flag when the gradient op is fixed.
-      bool check_not_have_gradient = true);
```

Comment:
>check_not_have_gradient [](start = 11, length = 23)

Latter when you have the Split/concat fixed, please update this to default true.

Reply:
will do